index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header class="bar-container">
+        <nav>
+           <img src="/Images/Light.png"/>
+           <img src="/Images/app_bar.png"/>
+        </nav>
```

Comment:
[OPTIONAL]

- Kindly remove this part from your webpage as they are added to the design to just represent that you are on the mobile view screen

![image](https://github.com/Arch-Noize/setup-and-mobile-first/assets/63473584/a43d77aa-f238-4505-86f2-c7e42736d0d2)